### PR TITLE
Initial tests for cli and api

### PR DESCRIPTION
**Pull request by flurmbo**
Initial tests requested in #70. Sorry if it's a bit messy, I have not tested with AVA before. I would appreciate any feedback on how I've written this code, especially if I should refactor out some of the code repeated in each test. 

Note that currently the test for the CLI sortAttribute flag is skipped because it is not yet passing (same test works fine for API).

### CODE REVIEW

File: packages/prettyhtml/test/api.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+test.cb('single quotes', t => {
+  const output = prettyhtml('<div id="foo" class=bar/>\n', {
+    singleQuote: true
+  })
+  t.is(
+    output.contents,
+    "<div id='foo' class='bar'/>\n",
+    'should use single instead of double quotes'
+  )
+  t.end()
```
Comment (StarpTech) [inline packages/prettyhtml/test/api.js]: t.end() is not needed in a synchronous test

File: packages/prettyhtml/test/cli.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }
+  )
+})
+
+test.skip.cb('sort attributes', t => {
+  fs.writeFile(
+    path.join(dir, 'sort-attributes.html'),
+    '<div b="bar" a="foo"/>\n',
+    err => {
+      if (err) throw err
```
Comment (StarpTech) [inline packages/prettyhtml/test/cli.js]: Use a assertion from ava

File: packages/prettyhtml/test/api.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const test = require('ava')
+const prettyhtml = require('../')
+
```
Comment (StarpTech) [inline packages/prettyhtml/test/api.js]: Could you add a test which only test the defaults? `prettyhtml test.html`

File: packages/prettyhtml/test/cli.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+const cliPath = path.join(__dirname, '../cli.js')
+
+try {
+  fs.mkdirSync(dir)
+} catch (err) {
+  if (err.code !== 'EEXIST') throw err
+}
+
+test.cb('single quotes', t => {
+  fs.writeFile(
```
Comment (StarpTech) [inline packages/prettyhtml/test/cli.js]: I would use writeFileSync

**Comment by StarpTech** [comment]
If you merge with master the `--skipAttributes` bug was fixed.

File: packages/prettyhtml/test/api.js (hunk 3 above)
Comment (flurmbo) [reply packages/prettyhtml/test/api.js]: Do you mean to test that, for example, prettyhtml('<div>Hello World</div>') returns '<div>Hello World</div>'?

File: packages/prettyhtml/test/cli.js (hunk 2 above)
Comment (flurmbo) [reply packages/prettyhtml/test/cli.js]: After switching to writeFileSync these assertions are no longer there. I do have some error handling when creating and deleting the tmp folder, should I use an ava assertion there?

File: packages/prettyhtml/test/api.js (hunk 3 above)
Comment (StarpTech) [reply packages/prettyhtml/test/api.js]: Yes, but with a more advanced example.

File: packages/prettyhtml/test/cli.js (hunk 2 above)
Comment (StarpTech) [reply packages/prettyhtml/test/cli.js]: Yes

**Comment by StarpTech** [comment]
@flurmbo Do you still working on this?

**Pull request closed without merging**